src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -186,6 +186,11 @@
 
+"OnboardingInfo_enableLoggingOfContactsPage_stateHeader" = "Status";
```

Comment:
Please also add some intermediate english translation as our translation folks might not have time to translate it

Reply:
Thanks for the catch.

Reply:
Done [here](https://github.com/corona-warn-app/cwa-app-ios/pull/517/commits/cf56b4302acaf9feacb5f652f441d17eed6f10ba).